stable/kibana/README.md

```diff
@@ -73,14 +73,15 @@ The following table lists the configurable parameters of the kibana chart and th
 | `dashboardImport.xpackauth.username`          | Optional Xpack username                    | `myuser`                               |
 | `dashboardImport.xpackauth.password`          | Optional Xpack password                    | `mypass`                               |
 | `dashboardImport.dashboards`                  | Dashboards                                 | `{}`                                   |
+| `plugins`                             | List of Kibana plugins to install                                 | None:                                   |
```

Comment:
can we add a small note here that the plugins should be URLs to zip files (which I understand to be the case from your commented example in the values.yaml)?